jerry-core/include/jerryscript-port.h

```diff
@@ -20,6 +20,32 @@
 #include <stdint.h>
 #include <stdio.h>
 
+#if defined(_MSC_VER)
```

Comment:
Somewhere it's `#if defined(_MSC_VER)`, somewhere else it's `#ifdef _MSC_VER`. The two are equivalent but it's best to keep a consistent style.

Reply:
BTW, I'd vote for the simpler form (`#ifdef _MSC_VER`), which is much more widely used in the existing code base than the longer form, which is mostly used when or-ing/and-ing of preprocessor predicates is necessary.

Reply:
I prefer defined because if we want to add expression then we modify less code

Reply:
But a consistent style is easier for the maintainers to maintain. (And, well, if we modify the predicate, we modify the same line(s), no matter which style we use, don't we?)

Reply:
OK， I hope reviews be more specific next time